### ISSUE TITLE
ATAG ARF Feedback: 2.3 Mobile Driving Licence - use case presents friction with ARF trust model

### ISSUE THREAD

**Issue by heatherdahl** [eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework#253]
### Description
**Name:** Heather Dahl, Indicio

**ARF Chapter:** 2.3 Mobile Driving Licence
_“A significant use case for the EUDI Wallet involves allowing Users to acquire, store, and present a mobile Driving Licence (mDL) as an attestation to prove their driving privileges.”_

**Recommendation:** The use case presented creates friction with the AFT trust model because it would require the registration of every officer and every device in the trust list. This is not realistic or practical for deployment at scale: to achieve adoption and scale, mDL should instead move to a certificate model.

**Comment by digeorgi** [comment]
Thank you for your input.
We believe that there might be a misunderstanding here. The trust model described in the ARF does not require all Relying Party Instances to be included in a Trusted List. Instead, after registration a Relying Party receives a certificate for each of its Relying Party Instances. This certificate is issued by a Relying Party Access Certificate Authority (CA). The trust anchors of this CA are included in a Trusted List. A Wallet Instance can use these trust anchors to verify the authenticity of the Relying Party Instance access certificates. This is described in section 6.4.2 of the main ARF document, as well as in Topics 27 and 31 of Annex 2.

So, in summary, the ARF already uses a certificate model, in accordance with your proposal.

**Comment by phin10** [comment]
Closed after release of ARF 1.10.0
